Assets/Scripts/Enemy/FinalBossController.cs

```diff
@@ -0,0 +1,129 @@
+using System.Collections;
+using Evolution;
+using UnityEngine;
+using UnityEngine.AI;
+using UnityEngine.UI;
+
+namespace Enemy
+{
+    public class FinalBossController : AbstractEnemyController
+    {
+        [SerializeField] private GameObject enemyBody;
+        [SerializeField] private GameObject projectile;
+        [SerializeField] private int projectileCountPerAttack = 6;
+        // Time between attacks in milliseconds
+        [SerializeField] private int timeBetweenAttacks = 5 * 1000;
+
+        private static readonly int MAX_ROLL_SPEED = 8;
+        private static readonly int MAX_NAV_SPEED = 2;        private static readonly int MAX_HP = 100;
+        private static readonly int SCORE = 1000;
```

Comment:
"Name 'MAX_ROLL_SPEED' does not match rule 'Static readonly fields (private)'. Suggested name is 'MaxRollSpeed'" (https://stackoverflow.com/a/242549)

În plus, pentru situația asta, `static readonly` poate fi simplificat în `const`.